### PR TITLE
chrony: add gnutls and pkg-config dependencies to enable NTS

### PR DESCRIPTION
**Pull request by lukevalenta**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/c/chrony.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Chrony < Formula
   url "https://chrony-project.org/releases/chrony-4.5.tar.gz"
   sha256 "19fe1d9f4664d445a69a96c71e8fdb60bcd8df24c73d1386e02287f7366ad422"
   license "GPL-2.0-only"
+  revision 1
```
[the remark this answers is not in the document]
Comment (lukevalenta) [reply Formula/c/chrony.rb:7]: > I don't think we should force an update on everyone

Makes sense -- I was unsure about whether a revision bump was needed. I'll remove it!

File: Formula/c/chrony.rb [hunk 2]
```diff
@@ -20,13 +20,14 @@ class Chrony < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "52eb2a90d0a86c0259c2bf09ae9474262adb420c5bc6dd75ecb0ff8b5124f1c9"
   end
 
+  depends_on "gnutls"
   depends_on "nettle"
+  depends_on "pkg-config"
```
Comment (cho-m) [inline Formula/c/chrony.rb:25]: Is `pkg-config` a runtime dependency?

Comment (lukevalenta) [reply Formula/c/chrony.rb]: I believe it's only a build dependency. I just updated to add the `=> :build` directive. Does that seem correct?

Comment (cho-m) [reply Formula/c/chrony.rb]: You'll need to move it above runtime dependencies but otherwise should be good.

**Review by carlocab — APPROVED** [review]

**Pull request merged**